src/user_config.yaml

```diff
@@ -2,12 +2,12 @@ config_validation:
   validate: True
   path: src/user_config_schema.yaml
 survey:
-  survey_type: "BERD"
+  survey_type: "PNP"
```

Comment:
can you remove this and the one below, we don't want to merge in this change.

Reply:
DONE